static/robots.txt

```diff
@@ -1,2 +1,10 @@
 User-agent: *
-Disallow:
+Disallow: */details/
+Disallow: */gallery/
+Disallow: */u/
+Disallow: */transfer/
```

Comment:
The url path of the transfer page is `/{prefix}/transfer` without the ending `/`

<img width="778" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/e764f606-af95-4ddf-89ef-82ae993d2c16">